docs/get-started/developers/installation.md

```diff
@@ -16,44 +16,76 @@ subnav:
+          "@lit/context": "https://www.redhatstatic.com/dssf-001/v2/@lit/context@{{ packageinfo.packages["node_modules/@lit/context"].version }}/development/index.js",
+          "@lit/reactive-element": "https://www.redhatstatic.com/dssf-001/v2/@lit/reactive-element@{{ packageinfo.packages["node_modules/@lit/reactive-element"].version }}/reactive-element.js",
+          "@lit/reactive-element/decorators/": "https://www.redhatstatic.com/dssf-001/v2/@lit/reactive-element@{{ packageinfo.packages["node_modules/@lit/reactive-element"].version }}/decorators/",
+          "@patternfly/pfe-core": "https://www.redhatstatic.com/dssf-001/v2/@patternfly/pfe-core@{{ packageinfo.packages["node_modules/@patternfly/pfe-core"].version }}/core.js",
+          "@patternfly/pfe-core/": "https://www.redhatstatic.com/dssf-001/v2/@patternfly/pfe-core@{{ packageinfo.packages["node_modules/@patternfly/pfe-core"].version }}/",
+          "@rhds/elements/lib/": "https://www.redhatstatic.com/dssf-001/v2/@rhds/elements@{{ pkg.version }}/lib/",
+          "@rhds/icons": "https://www.redhatstatic.com/dssf-001/v2/@rhds/icons@{{ packageinfo.packages["node_modules/@rhds/icons"].version }}/icons.js",
+          "@rhds/icons/": "https://www.redhatstatic.com/dssf-001/v2/@rhds/icons@{{ packageinfo.packages["node_modules/@rhds/icons"].version }}/",
+          "@rhds/tokens/media.js": "https://www.redhatstatic.com/dssf-001/v2/@rhds/tokens@{{ packageinfo.packages["node_modules/@rhds/tokens"].version }}/js/media.js",
+          "@rhds/tokens/": "https://www.redhatstatic.com/dssf-001/v2/@rhds/tokens@{{ packageinfo.packages["node_modules/@rhds/tokens"].version }}/",
+          "lit": "https://www.redhatstatic.com/dssf-001/v2/lit@{{ packageinfo.packages["node_modules/lit"].version }}/index.js",
+          "lit/": "https://www.redhatstatic.com/dssf-001/v2/lit@{{ packageinfo.packages["node_modules/lit"].version }}/",
+          "lit-element/lit-element.js": "https://www.redhatstatic.com/dssf-001/v2/lit-element@{{ packageinfo.packages["node_modules/lit-element"].version }}/lit-element.js",
+          "lit-html": "https://www.redhatstatic.com/dssf-001/v2/lit-html@{{ packageinfo.packages["node_modules/lit-html"].version }}/lit-html.js",
+          "lit-html/": "https://www.redhatstatic.com/dssf-001/v2/lit-html@{{ packageinfo.packages["node_modules/lit-html"].version }}/",
+          "tslib": "https://www.redhatstatic.com/dssf-001/v2/tslib@{{ packageinfo.packages["node_modules/tslib"].version }}/tslib.es6.mjs"
+      }
     }
   }
 </script>
 ```
 
-Once the import map is established, you can load the element with the following 
-module, containing a [bare module specifier][barespec]. The example below shows 
-how you'd load in <`rh-button>`.
+### Loading individual elements
+
+Once the import map is established, you can load individual elements using a [bare module specifier][barespec].
```

Comment:
can we get a line here explaining that imports are deduplicated? like "it's ok to import an element more than once on any given page, as long as it's always imported using the same bare specifier"